src/App.tsx

```diff
@@ -1,26 +1,160 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useMemo, useState } from 'react';
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+import { Errors } from './types/ErrorType';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoHeader } from './components/TodoHeader';
+import { ErrorNotification } from './components/ErrorNotification';
+import {
+  getTodos,
+  addTodo,
+  USER_ID,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<FilterType>(FilterType.All);
+  const [errorMessage, setErrorMessage] = useState<Errors>(Errors.Default);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadingTodo, setLoadingTodo] = useState<number[]>([]);
```

Comment:
syntax error, cuz i see an array `number[]`
```suggestion
  const [loadingTodos, setLoadingTodos] = useState<number[]>([]);
```